base/frontend/sourcegraph-frontend.ConfigMap.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion:  v1
+kind: ConfigMap
```

Comment:
Are we keeping this change in for this PR? If so, I will add it to the overlays. Otherwise it needs to be removed and the env values added to the initContainer. (Happy to do either, just need a decision for this PR)